src/main/java/com/gmail/nossr50/api/exceptions/McMMOPlayerNotFoundException.java

```diff
@@ -1,11 +1,13 @@
 package com.gmail.nossr50.api.exceptions;
 
+import javax.annotation.Nonnull;
+
 import org.bukkit.entity.Player;
 
 public class McMMOPlayerNotFoundException extends RuntimeException {
     private static final long serialVersionUID = 761917904993202836L;
 
-    public McMMOPlayerNotFoundException(Player player) {
+    public McMMOPlayerNotFoundException(@Nonnull Player player) {
```

Comment:
We are not using javax annotations, please keep the null/nullable annotations consistent with the rest of the code base.
We are using jetbrains library for annotations. It's also a good idea to avoid using javax because I believe that is oracle specific and doesn't ship out with adoptopenjdk, I could be wrong though.

Reply:
My bad with the annotations, I'll change them.
And javax is not oracle-specific, javax annotations are part of JSR305 from the Google Findbugs project, it is included in the spigot executable jar, so nothing platform-specific at all.

I took a look at the `pom.xml` tree and found this dependency to be inherited from `worldedit-core`, I usually exclude unwanted or conflicting libraries like that in the pom. I think it would help avoid mistakes like this in the future since jsr305 annotations aren't gonna be available during development at all.
What's your opinion about this?

This would be the relevant pom entry I would add to the worldedit dependency:
```xml
<!-- ... -->
            <exclusions>
                <exclusion>
                    <!-- We use jetbrains instead. Excluding this -->
                    <!-- prevents us from using inconsistent annotations -->
                    <groupId>com.google.code.findbugs</groupId>
                    <artifactId>jsr305</artifactId>
                </exclusion>
            </exclusions>
```

Reply:
You can add the exclusion

Reply:
I may have been thinking of JavaFX and not javax